tests/formats/dataclass/parsers/test_utils.py

```diff
@@ -26,6 +26,28 @@ def test_parse_value(self, mock_to_python):
         self.assertTrue(2, ParserUtils.parse_value([int], "1", None))
         mock_to_python.assert_called_once_with([int], "1", None)
 
+    def test_parse_value_with_tokens_true(self):
+        actual = ParserUtils.parse_value([int], " 1 2 3", list, None, True)
+        self.assertEqual([1, 2, 3], actual)
+
+        actual = ParserUtils.parse_value([int], ["1", "2", "3"], list, None, True)
+        self.assertEqual([1, 2, 3], actual)
+
+    @mock.patch("xsdata.formats.dataclass.parsers.utils.to_python", return_value=2)
+    def test_parse_value_with_ns_map(self, mock_to_python):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/tefra/xsdata/pullRequest?prid=5343831)